src/routes/homepage/sections/whatIsTLIP.svelte

```diff
@@ -18,9 +18,9 @@
                     <img src="./assets/{image}" alt={title} class="image-shadow">
                 </div>
                 <div class="w-full xl:w-7/12 md:{idx % 2 ? 'pr': 'pl'}-28 flex flex-col md:items-start items-center justify-center text-center md:text-left">
-                    <h4 class="font-bold text-15 lg:text-20 tracking-0.02 leading-6 capitalize lg:text-left">{eyebrow}</h4>
-                    <p class="font-bold text-24 lg:text-32 tracking-normal lg:tracking-0.02 leading-7 md:leading-10 md:my-3 my-2">{title}</p>
-                    <p class="font-normal text-12 lg:text-16 leading-6 tracking-0.04 ">{description}</p>
+                    <h4 class="font-bold text-15 lg:text-20 tracking-0.02 leading-6 capitalize lg:text-left text-grey-500">{eyebrow}</h4>
+                    <h3 class="text-24 lg:text-32 tracking-normal lg:tracking-0.02 md:my-3 my-2">{title}</h3>
```

Comment:
It is not necessary to have them (`text-24` and `lg:text-32`)
![image](https://user-images.githubusercontent.com/72078237/173345039-9bee360e-2348-4df2-8a24-e93674d92e9f.png)

I change it, don't worry :wink: